projects/odf_transform/odf_transform/odf_to_netcdf.py

```diff
@@ -24,13 +24,20 @@ def read_config(config_file):
         config = json.load(fid)
 
         # Read Vocabulary file
+        config["vocabulary"] = {}
         for vocab_file in config["vocabularyFileList"]:
-            config.update({"vocabulary": {}})
-            with open(vocab_file) as fid:
-                vocab = json.load(fid)
-            config["vocabulary"].update(vocab)
+            if vocab_file.endswith('json'):
```

Comment:
can we remove json support if we're swtiching to CSV, just to simplify

Reply:
let's remove the json option

Reply:
I'll also make it only expect only a single vocabulary csv file